components/Votes/Votes.tsx

```diff
@@ -405,51 +441,74 @@ export function Votes() {
             ) : null}
           </ButtonInnerWrapper>
         </ButtonOuterWrapper>
-      ) : null}
-      {determineVotesToShow().length > defaultResultsPerPage && (
-        <PaginationWrapper>
-          <Pagination
-            paginateFor="activeVotesPage"
-            numberOfEntries={determineVotesToShow().length}
+        {pv ? (
+          <>
+            <Divider />
+            {pv}
+          </>
+        ) : null}
+      </>
+    );
+  }
+  function upcomingView() {
```

Comment:
not important but interesting — these don't actually have to be functions if they aren't taking props. you can just do `const upcomingView = (<>jsx stuff<>)` and put it in the other jsx the same as you did here like `{upcomingView}`

Reply:
probably true, i am composing one of them though, but the others probably dont have to be